content/webapp/pages/search/events.tsx

```diff
@@ -98,9 +80,24 @@ export const EventsSearchPage: NextPageWithLayout<Props> = ({
                 </span>
 
                 <SortPaginationWrapper>
-                  <Sort
+                  {/* TODO */}
```

Comment:
Why is this a TODO?

Reply:
Ah just because we need to re-add it, Georgia is working on it atm. Agreed it could've been clearer.

Reply:
(now amended to clarify)